components/library/poppler/Makefile

```diff
@@ -21,31 +21,30 @@ USE_DEFAULT_TEST_TRANSFORMS= yes
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		poppler
-COMPONENT_VERSION=	24.3.0
-HUMAN_VERSION=	24.03.0
+COMPONENT_VERSION=	24.11.0
+#HUMAN_VERSION=	24.09.0
 COMPONENT_SUMMARY=	Poppler is a PDF rendering library based on the xpdf-3.0 code base
 COMPONENT_PROJECT_URL=	https://poppler.freedesktop.org
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(HUMAN_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
-COMPONENT_ARCHIVE_HASH=	sha256:bafbf0db5713dec25b5d16eb2cd87e4a62351cdc40f050c3937cd8dd6882d446
+COMPONENT_ARCHIVE_HASH=	sha256:7723d880565211740c13649d24a300257b86ddd7fa2d208187ff7e5cc8dfbd58
 COMPONENT_ARCHIVE_URL=	$(COMPONENT_PROJECT_URL)/$(COMPONENT_ARCHIVE)
 COMPONENT_FMRI=		library/libpoppler
 COMPONENT_CLASSIFICATION=	Desktop (GNOME)/Libraries
 COMPONENT_LICENSE=	GPLv2
 COMPONENT_LICENSE_FILE=	COPYING
 
+COMPONENT_NAME_1=   testfiles
+GIT_REPO_1=			https://gitlab.freedesktop.org/poppler/test.git
+GIT_COMMIT_ID_1=    ff3133cd
+
+
 include $(WS_MAKE_RULES)/common.mk
 
 PATH= $(QT5_BINDIR):$(PATH.gnu)
-PKG_CONFIG_PATH= $(PKG_CONFIG_PATH.$(BITS)):$(QT5_PKG_CONFIG_PATH)
+PKG_CONFIG_PATH += $(QT5_PKG_CONFIG_PATH)
```

Comment:
I think `PKG_CONFIG_PATH` is not designed to work properly with `+=`.  You probably should use `PKG_CONFIG_PATH.prepend` instead.